### PR TITLE
Add logstash as a marathon job

### PR DESCRIPTION
**Pull request by Dgadavin**
- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release
- [ ] Updates documentation relevant to the changes

Add a new logstash role as a marathon job and added it to addons. Tested with kafka and elasticsearch.

### CODE REVIEW

**Comment by siddharthist** [comment]
@Dgadavin Thanks for the PR! Pinging @ryane for review.

(don't worry if your CI build fails, we're working on it - #1760)

**Comment by ryane** [comment]
can you add a readme that describes what this role is, what you would use it for, how to install it (it probably needs an install playbook in the `/addons` directory), what it depends on, how to customize it, etc? you can see our other role readmes for examples.

File: addons/pangea.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Install Pangea on top of MANTL
+
+# CHECK SECURITY - when customizing you should leave this in. If you take it out
+# and forget to specify security.yml, security could be turned off on components
+# in your cluster!
+- include: "{{ playbook_dir }}/../playbooks/check-requirements.yml"
+
+- hosts: role=edge
+  roles:
```
Comment (ryane) [inline addons/pangea.yml]: would it make more sense for the pangea install to live in another repo?

File: roles/logstash-marathon/files/logstash.conf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    remove_field => ["type","beat"]
+  }
+}
+output {
+  elasticsearch {
+    codec => "json"
+    hosts => [ "elasticsearch-client-mantl.service.consul:9200" ]
+    index => "logs-%{+YYYY.MM.dd}"
+  }
+}
```
Comment (ryane) [inline roles/logstash-marathon/files/logstash.conf]: inputs and outputs should be configurable

File: roles/pangea/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (ryane) [inline roles/pangea/tasks/main.yml]: I am not sure what the intention is here but this is not going to be compatible across cloud providers. the default ssh user and hostnames will vary. Also, mantl-pangea is a private repository. This should really live somewhere else.

**Comment by ryane** [comment]
I am wondering if this and #1784 should be combined. Also, the pangea parts of this don't really belong in the mantl repository as far as I can tell.

**Comment by Dgadavin** [comment]
@ryane All files about Pangea is not for this PR. This is my fault. Maybe I will close this PR and open another.

**Comment by Dgadavin** [comment]
Closed because clone of #1784

**Pull request closed without merging**